modules/ROOT/pages/_partials/common-releasenotes.adoc

```diff
@@ -23,6 +23,92 @@ include::{root-partials}block-abstract.adoc[]
 
 // END DO NOT REMOVE
 
+[#lbl-rel282]
+== Release 2.8.2 (March 2021)
+// ====
+// <<new-features-282>> *|* <<improvements-282>> *|* <<issues-and-resolutions-282>>
+// ====
+// tag::feature-highlights-282[]
+// tag::new-282-all[]
+
+[#new-features-282]
+// === New Features
+
+// tag::features-282[]
+// Metrics REST API::
+// end::features-282[]
+// end::new-282-all[]
+// end::feature-highlights-282[]
+
+[#improvements-282]
+// tag::other-enhancements-282[]
+// tag::other-enhancements-highlights-282[]
+// end::other-enhancements-highlights-282[]
+// end::other-enhancements-282[]
+
+
+[#issues-and-resolutions-282]
+=== Issues and Resolutions
+// tag::issue-resolutions-282[]
+
+// tag::issue-resolutions-summary-282[]
+Release 2.8.2 fixes an issue that could result in a loop when creating a mobile tombstone, if a Couchbase Server tombstone already exists for that key.
```

Comment:
maybe, "...could result in Sync Gateway going into an infinite loop when creating..."

Reply:
addressed https://github.com/couchbase/docs-sync-gateway/pull/431/commits/8c2661e7f7a1bfc73c8589f823688d2873196d76